components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/listener/ConsoleAuthorizedAPIListener.java

```diff
@@ -153,10 +154,22 @@ public AuthorizedAPI postGetAuthorizedAPI(AuthorizedAPI authorizedAPI, String ap
     private String getConsoleAppId(String tenantDomain) throws IdentityApplicationManagementException {
 
         ApplicationManagementService applicationManagementService = ApplicationManagementService.getInstance();
-        return applicationManagementService.getApplicationResourceIDByInboundKey("CONSOLE",
+        String consoleInboundKey = buildConsoleInboundKey(tenantDomain);
+
+        return applicationManagementService.getApplicationResourceIDByInboundKey(consoleInboundKey,
                 "oauth2", tenantDomain);
     }
 
+    private String buildConsoleInboundKey(String tenantDomain) {
+
+        String consoleInboundKeyDefault = "CONSOLE";
+        if (IdentityTenantUtil.isTenantQualifiedUrlsEnabled() || tenantDomain.equals("Carbon.super")) {
```

Comment:
```suggestion
        if (IdentityTenantUtil.isTenantQualifiedUrlsEnabled() || ("carbon.super").equalsIgnoreCase(tenantDomain)) {
```

Does `Carbon.super` needs to be case sensitive?
and also can we move these constants to a Constants file if available?

Reply:
This is fixed with 0c4ad2afe4568c5d6101e90e071f103d30058a06